sys/net/gnrc/network_layer/ipv6/static_addr/gnrc_ipv6_static_addr.c

```diff
@@ -117,6 +117,10 @@ void auto_init_gnrc_ipv6_static_addr(void)
     gnrc_netif_t *upstream = NULL;
     gnrc_netif_t *downstream = NULL;
 
+    if (gnrc_netif_highlander() || gnrc_netif_numof() == 1) {
+        upstream = gnrc_netif_iter(NULL);
+    }
+
```

Comment:
There is still an else needed for the while loop not to be compiled in. Or does an early return here make sense?

Reply:
Ah good point